content/cpp/concepts/math-functions/terms/acos/acos.md

```diff
@@ -37,3 +37,21 @@ int main() {
   // Output: 1.5708 radians
 }
 ```
+
+## Codebyte Example
+
+The following example is runnable and outputs the inverse cosine of `-0.5`:
+
+```codebyte/cpp
+#include <iostream>
+#include <cmath>
```

Comment:
may want to include std here and for other examples so we don't have to keep repeating std